Source/CubicHorndeski/CubicHorndeski.impl.hpp

```diff
@@ -148,43 +179,37 @@ void CubicHorndeski<coupling_and_potential_t>::pre_compute_no_gauge(
     data_t numerator = 0.;
     data_t denominator = 0.;
 
-    ////////////////////////////////////////////////////////////////////////
-    // DEFINE VARIABLES USED REPETEDLY
-
-    data_t chi = vars.chi;
-    data_t Pi = vars.Pi;
-
-    E.Pi2 = Pi * Pi;
-
     // dphi_dot_dphi - not conformal
     data_t dphi_dot_dphi = 0.;
-    FOR(i, j) { dphi_dot_dphi += h_UU[i][j] * d1.phi[i] * d1.phi[j]; }
-    dphi_dot_dphi *= chi;
+    FOR(i, j)
+    {
+        dphi_dot_dphi += vars.chi * h_UU[i][j] * d1.phi[i] * d1.phi[j];
+    }
 
     // X
```

Comment:
This looks funny as a comment! Maybe expand to say X is the kinetic term in the Lagrangian or something like that.